src/Command/DownCommand.php

```diff
@@ -31,7 +31,7 @@
  * yii migrate:down all # revert all migrations
  * ```
  */
-#[AsCommand('migrate:down', 'Downgrades the application by reverting old migrations.')]
+#[AsCommand('migrate:down', 'Reverts the specified number of latest migrations.')]
```

Comment:
I have changed strings but did not adjust tests.

Reply:
done.